test/representers/api/paged_collection_representer_test.rb

```diff
@@ -52,4 +43,15 @@
     representer.href_url_helper({foo: 1, bar: 2, camp: 3}).must_equal "this is a test"
   end
 
+  describe 'test with routing' do
+
+    before { define_routes }
+    after { Rails.application.reload_routes! }
+
+    it 'gets a route url helper method with parent' do
+      representer.represented.options[:parent] = TestParent.new(1, true)
+      representer.represented.options[:item_class] = TestObject
+      representer.href_url_helper({page: 1}).must_equal "/api/test_parent/1/test_objects?page=1"
```

Comment:
Line is too long. [96/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.